fury/shaders/base.py

```diff
@@ -4,10 +4,16 @@
 from fury.lib import (numpy_support, Command, VTK_OBJECT,
                       calldata_type, DataObject, Shader)
 
-SHADERS_TYPE = {"vertex": Shader.Vertex,
-                "geometry": Shader.Geometry,
-                "fragment": Shader.Fragment,
-                }
+from os.path import join as pjoin, dirname, isfile
```

Comment:
pep8:  import order.   see https://peps.python.org/pep-0008/#imports for more information

Reply:
This should be fixed, but PTAL.